core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -1232,24 +1243,16 @@ private[ui] class TaskPagedTable(
 
   override def pageLink(page: Int): String = {
```

Comment:
Yeah, it technically works because browsers are pretty resilient but it's a little ugly when reading the generated HTML.